### PR TITLE
feat(3102): add FAQ page

### PR DESCRIPTION
**Pull request by VGau**
### Checklist

* [ ] I wrote new tests for my new core changes.
* [ ] I have successfully ran tests, style checker and build against my new changes locally.
* [ ] I have informed the team of any breaking changes if there are any.

### CODE REVIEW

File: bridge-ui/src/app/faq/page.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+          <p>
+            Note that this only speeds up your submission of the bridge transaction. It does not actually speed up the
+            bridging process itself that you initiate with this transaction — you cannot speed up the 8-32 hour waiting
+            time for a withdrawal.
+          </p>
+        </Collapse>
+
+        <Collapse title="Where can I access support?">
+          <p>Click the “Contact support” button in the bottom-left corner of the Linea Bridge app.</p>
+          <p>Alternatively, head to the Linea Help Center and click the “Contact” button on the homepage.</p>
```
Comment (thedarkjester) [inline bridge-ui/src/app/faq/page.tsx:196]: why make the user go back to the homepage or hunt for a button to Contact - could we not link here?

File: bridge-ui/src/app/faq/page.tsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            .
+          </p>
+        </Collapse>
+
+        <Collapse title="How long does the bridge take?">
+          <p>This depends on the direction of your bridge:</p>
+          <ul className="list-disc p-4">
+            <li>Deposit (L1 -&gt; L2): Approximately 20 minutes.</li>
+            <li>
+              Withdrawal (L2 -&gt; L1): Between 8 and 32 hours. The transaction must be finalized on Ethereum Mainnet
```
Comment (thedarkjester) [inline bridge-ui/src/app/faq/page.tsx]: should this not be `The L2 transaction` for clarity

File: bridge-ui/src/app/faq/page.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+              target="_blank"
+              className="link link-primary"
+            >
+              Linea help center
+            </Link>
+            .
+          </p>
+        </Collapse>
+
+        <Collapse title="How long does the bridge take?">
```
Comment (thedarkjester) [inline bridge-ui/src/app/faq/page.tsx]: The `bridge` should be `bridging` I think

File: bridge-ui/src/app/faq/page.tsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+              target="_blank"
+              className="link link-primary"
+            >
+              MetaMask Portfolio bridge
+            </Link>
+            .
+          </p>
+        </Collapse>
+
+        <Collapse title="I can't see the token I want to bridge">
```
Comment (thedarkjester) [inline bridge-ui/src/app/faq/page.tsx]: this isn't phrased as a question  - `Why can't I see the token I want to bridge?`

File: bridge-ui/src/app/faq/page.tsx [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+              target="_blank"
+              className="link link-primary"
+            >
+              liquidity fragmentation
+            </Link>{" "}
+            and mitigates the risk of funds loss.
+          </p>
+        </Collapse>
+
+        <Collapse title="I can't see my bridged tokens in my wallet">
```
Comment (thedarkjester) [inline bridge-ui/src/app/faq/page.tsx]: same thing re: question vs. statement

**Review by thedarkjester — APPROVED** [review]

**Review by thedarkjester — APPROVED** [review]

**Review by Julink-eth — APPROVED** [review]

**Review by count-sum — APPROVED** [review]

**Pull request merged**